lambda_utils/src/wellcome_lambda_utils/s3_utils.py

```diff
@@ -1,5 +1,7 @@
 # -*- encoding: utf-8 -*-
 
+import dateutil.parser
```

Comment:
You need to add _dateutil_ to the _install_requires_ for the common package.

Reply:
👍